src/main/java/am/itspace/jobboard/config/SecurityConfig.java

```diff
@@ -0,0 +1,49 @@
+package am.itspace.jobboard.config;
+
+import am.itspace.jobboard.entity.enums.UserRole;
+import am.itspace.jobboard.security.UserDetailService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@RequiredArgsConstructor
+public class SecurityConfig {
+
+    private final PasswordEncoder passwordEncoder;
+    private final UserDetailService userDetailService;
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity httpSecurity) throws Exception {
+        httpSecurity
+                .csrf().disable()
+                .authorizeHttpRequests()
+                .requestMatchers("/").permitAll()
+                .requestMatchers("/admin/**").hasAuthority(UserRole.ADMIN.name())
+                .requestMatchers("/login/page").permitAll()
```

Comment:
Change login and register requestMatchers, make them`

.requestMatchers("/login/**").anonymous()
and
.requestMatchers("/register/**").anonymous()